src/Schema.php

```diff
@@ -20,7 +20,7 @@ class Schema {
    * CORE_STANDARDS_HTTP_HEADERS.
    */
   const HTTP_RESPONSE_HEADERS = [
-    'Content-Security-Policy' => '"frame-ancestors"',
+    'Content-Security-Policy-Report-Only' => 'frame-ancestors \'self\'',
```

Comment:
The wrapping double quotes are missing.
```suggestion
    'Content-Security-Policy-Report-Only' => '"frame-ancestors \'self\'"',
```

However, the long-term consequence of adding this line will be that a site cannot remove it or replace it with an actual CSP (without "-Report-Only").  Due to the array_merge(), the defaults are always applied.  I don't know what happens if both a CSP and a CSPRO are sent, and I don't think it makes sense to find out.

Yes, we could add support for setting NULL values as a way to "unset" the CSPRO.  But at that point I would say the configuration of our solution would borderline become more complicated than the headers themselves.

As a result, none of the solutions seem mature enough to be distributed to all of our projects currently.

Based on these grounds, I would advise to move forward without the CSP/CSPRO at all for now:
```suggestion
```

Instead, we should create a follow-up task to investigate whether we can automatically generate a safe(r) Content Security Policy for all of our sites — or whether we need to introduce a different way to generate it.

(With regard to different ways, for example, we could have a crawler that checks all pages of a website to collect all the domains that are used by fonts, scripts, styles, media, and frames, so we can review them and compose a corresponding CSP.)